cypress/unit/store-mutation-setHeuristic.cy.js

```diff
@@ -0,0 +1,31 @@
+import { mutations } from "~/store";
+
+let state = {};
+
+describe("setHeuristic", () => {
+
+    beforeEach(() => {
+
+        // Setup
+        state = {
+
+            dataDictionary: {
+                annotated: {
+                    column1: { transformationHeuristic: "" }
```

Comment:
minor point: in the default state of the `store.state` the `transformationHeuristic` attribute doesn't exist for a column. 
That makes sense, not every column will need one, so probably best not to create the field by default. 
However: it would be good to have a test here that checks that we actually successfully create the attribute as well.
i.e.: 

- mock the state to not have `transformationHeuristic` (i.e. to reflect the default state), 
- fire the mutation
- ensure that 1) the respective column now does have the `transformationHeuristic` attribute, and 2) that the value of this attribute is correct

Reply:
@surchs Makes sense. See what you think about my new commit.

Reply:
@surchs Okay. Just added the existence check as well. Check it out and see what you think.